includes/payment-methods/class-wc-gateway-stripe-bancontact.php

```diff
@@ -205,7 +205,7 @@ public function payment_fields() {
 	public function create_source( $order ) {
 		$currency                = WC_Stripe_Helper::is_wc_lt( '3.0' ) ? $order->get_order_currency() : $order->get_currency();
 		$order_id                = WC_Stripe_Helper::is_wc_lt( '3.0' ) ? $order->id : $order->get_id();
-		$return_url              = $this->get_stripe_return_url( $order );
+		$return_url              = wp_sanitize_redirect( $this->get_stripe_return_url( $order ) );
```

Comment:
In addition to bancontact and ideal, others use this too - maybe we should be running wp_sanitize_redirect inside get_stripe_return_url instead?